openvdb_points/tools/PointDataGrid.h

```diff
@@ -439,12 +471,13 @@ class PointDataLeafNode : public tree::LeafNode<T, Log2Dim> {
         return totalPointCount;
     }
 
-    void deactivateEmptyVoxels()
+    /// @brief Uses the point count of each voxel to update the active voxel mask on this
+    /// leaf node. Voxels with a point count of zero will be deactivated, and voxels with
+    /// a non-zero point count will be activated.
+    void updateActiveVoxelMask()
```

Comment:
My rationale here was to explicitly name the entity we're updating (e.g. we are updating the entire mask, not just the active voxels). Perhaps following the naming used on the base leaf methods (setValueMask, setValueMaskOn, setValueMaskOff) would be more consistent, e.g. 'updateValueMask'?